changelog/1512.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+The `~plasmapy.analysis` and `~plasmapy.dispersion` subpackages are now
+directly imported in `plasmapy.__init__`.
```

Comment:
Wrong tense...

```suggestion
Exposed the `~plasmapy.analysis` and `~plasmapy.dispersion` subpackages to the
`plasmapy` namespace..
```

Reply:
Good catch on this.  I'm at a loss for how this has been missed for so long.

Reply:
It is indeed import-ant.  😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹 😹

Reply:
You made the rules, I'm only just enforcing them. 😆